net.sonobus.SonoBus.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: net.sonobus.SonoBus
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '20.08'
+command: sonobus
+#rename-desktop-file: sonobus.desktop
+rename-icon: 
```

Comment:
rename-icon need an icon name. But since you install it manually, you should remove this here.

Reply:
This is why your build fails

Reply:
Thanks, I miss that.